alicloud/resource_alicloud_router_interface_test.go

```diff
@@ -94,8 +94,12 @@ resource "alicloud_vpc" "foo" {
   cidr_block = "172.16.0.0/12"
 }
 
+data "alicloud_regions" "current_regions" {
+  current = true
+}
+
 resource "alicloud_router_interface" "interface" {
-  opposite_region = "cn-beijing"
+  opposite_region = "${data.alicloud_regions.current_regions.regions.0.id}"
```

Comment:
I think this region can not be modified. At present, not all region can support creating router interface automatically by terraform because of law